### PR TITLE
Update package.json: name, repository and metadata

### PR DESCRIPTION
**Pull request by jhermsmeier**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
   "main": "index.js",
+  "gypfile": true,
   "scripts": {
     "configure": "node-gyp configure",
     "build": "node-gyp build",
     "test": "mocha",
     "publish-please": "publish-please",
-    "prepublish": "publish-please guard"
+    "prepublish": "publish-please guard",
+    "install": "node-gyp rebuild"
```
Comment (petrosagg) [inline package.json]: why is this necessary?

File: package.json [hunk 2]
```diff
@@ -1,16 +1,18 @@
 {
-  "name": "node-ext2fs",
+  "name": "ext2fs",
   "version": "0.0.2",
   "description": "NodeJS native bindings to the libext2fs for cross-platform ext filesystem handling",
   "author": "Petros Angelatos <petrosagg@resin.io>",
   "license": "Apache-2.0",
   "main": "index.js",
+  "gypfile": true,
```
Comment (petrosagg) [inline package.json]: why is this necessary?

File: package.json (hunk 1 above)
Comment (jhermsmeier) [inline package.json]: It runs `clean`, `configure` and `build`, and is usually what you'd want for dependants installing this (so that it cleans & reconfigures when it's updated)

File: package.json (hunk 2 above)
Comment (jhermsmeier) [inline package.json]: It's not necessary, but [mentioned](https://github.com/nodejs/node-addon-examples/tree/master/1_hello_world#step-3) that it should be added, so that npm knows it's a binary module without having to guess based on package contents.

File: package.json (hunk 1 above)
Comment (petrosagg) [inline package.json]: hm ok. It seems to be happening automatically when I do npm install.
